packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx

```diff
@@ -85,12 +85,12 @@ function CardCategory({
       title={item.label}
       description={translate(
         {
-          message: '{count} items',
+          message: '({count} items) {description}',
           id: 'theme.docs.DocCard.categoryDescription',
           description:
             'The default description for a category card in the generated index about how many items this category includes',
         },
-        {count: item.items.length},
+        {description: item.description ?? '', count: item.items.length},
```

Comment:
hmmm, not really like this fallback to '', wonder if we shouldn't do this instead:

```jsx
desctiption={item.description ?? translate({...}}
```

Category items without descriptions will keep the previous label

Does it make sense?